src/mintpy/utils/utils0.py

```diff
@@ -318,8 +317,8 @@ def utm2latlon(meta, easting, northing):
     """
```

Comment:
likewise (i cant comment directly on the line) it looks like theres `lat, lon` as parameter descriptions here?